plugins/generator-1.20.1/forge-1.20.1/mappings/projectiles.yaml

```diff
@@ -1,40 +1,52 @@
 _default:
   - Snowball
   - EntityType.SNOWBALL
+  - Items.AIR
 _mcreator_prefix: "CUSTOM:"
 _mcreator_map_template:
   - "@NAMEEntity"
   - "@JavaModNameEntities.@REGISTRYNAME.get()"
+  - "@NAMEEntity.PROJECTILE_ITEM.getItem()"
 Arrow:
   - Arrow
   - EntityType.ARROW
+  - Items.ARROW
 DragonFireball:
   - DragonFireball
   - EntityType.DRAGON_FIREBALL
+  - Items.AIR
 Egg:
   - ThrownEgg
   - EntityType.EGG
+  - Items.AIR
 EnderPearl:
   - ThrownEnderpearl
   - EntityType.ENDER_PEARL
+  - Items.AIR
 ExperienceBottle:
   - ThrownExperienceBottle
   - EntityType.EXPERIENCE_BOTTLE

```

Comment:
This one lacks 3rd item

Reply:
https://github.com/MCreator/MCreator/pull/3456#discussion_r1252240648